lib/eventRouter.ts

```diff
@@ -104,7 +104,7 @@ const processInOrder = async (events: any, routerRules: any) => {
     } else {
       const promise = Promise.all(handlers.map((handler: any) => handler(event).catch(handleError)));
       const res = await promise;
-      logger.debug(logPrefix, `Event handlers processed event ${event.id} with result: [ ${res.join(' | ')} ]`);
+      logger.debug(logPrefix, `Event handlers processed event ${event.id} with result: ${res}`);
```

Comment:
I think this will still result in `[Object object]`. You should either `JSON.stringify` it or pass it as a separate argument to `logger.debug`:
```
logger.debug(logPrefix, `Event handlers processed event ${event.id} with result:`, res)
```